src/app/service/poap/StartPOAP.ts

```diff
@@ -350,5 +354,5 @@ const startTwitterPOAPFlow = async (ctx: CommandContext, guildMember: GuildMembe
 	if (result.ok != 1) {
 		await guildMember.send('One more thing, there was a problem setting counting you as a participants.. You just might miss the POAP, sorry.');
 	}
-	Log.debug('POAP Twitter spaces event tracking message sent');
+	Log.debug('POAP Twitter spaces event event start message sent');
```

Comment:
Did you purposely put two "events" here? I know it's only for logging, but just want to be sure this was intentional.

Reply:
dam I'm too young to be stroking lol